sel/src/Sel/Hashing/SHA2/SHA512.hs

```diff
@@ -179,7 +180,7 @@ hashText text = hashByteString (Text.encodeUtf8 text)
 -- ...   SHA512.finaliseMultipart multipartState
 --
 -- @since 0.0.1.0
-newtype Multipart = Multipart (Ptr CryptoHashSHA512State)
+newtype Multipart s = UnsafeMultipart (Ptr CryptoHashSHA512State)
```

Comment:
I don't think you need to rename the data constructor because we're not exporting it. :)

Reply:
ah yeah, I just thought that even though we're not currently it would be nice to make it clear that usage of this constructor is generally unsafe.